package.json

```diff
@@ -18,7 +18,7 @@
     "Richard Chamberlain <richard_chamberlain@uk.ibm.com> (https://github.com/rnchamberlain)"
   ],
   "scripts": {
-    "test": "tap test/test*.js"
+    "test": "make lint && tap test/test*.js"
```

Comment:
Requiring `make` breaks `npm test` on Windows:

```
C:\work\node\github\nodereport>npm test

> nodereport@1.0.7 test C:\work\node\github\nodereport
> make lint && tap test/test*.js

'make' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.

C:\work\node\github\nodereport>
```

I would prefer, if possible, that we are able to run the linter on all supported platforms. It seems from https://github.com/nodejs/node/issues/5514#issuecomment-191188708 that in the main Node.js project `cpplint` is not currently run on Windows, but it sounds more like a tooling problem that an issue with `cpplint.py` itself. Prior to this PR we don't have explicit `makefile`/`vcbuild.bat` since we rely on `binding.gyp` and `node-gyp` -- Perhaps downloading and running cpplint could be a script written in Node.js?